### PR TITLE
CP-1062 eslint, karma-jspm

### PR DESCRIPTION
**Pull request by valotas**
Make use of basic [eslint](http://eslint.org/) rules and fixed warnings

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "rules": {
+        "indent": [
```
Comment (trentgrover-wf) [inline .eslintrc]: can you make this the default 4 space indent (more consistent with other OSS JS repos from our org)?

**Comment by valotas** [comment]
Done

File: src/adapter.js [hunk 2]
```diff
@@ -39,14 +42,14 @@
...
-        
+
         // Load everything specified in loadFiles
         for (var i = 0; i < karma.config.jspm.expandedFiles.length; i++) {
             var modulePath = karma.config.jspm.expandedFiles[i];
             var promise = System['import'](extractModuleName(modulePath))
-                ['catch'](function(e) {
-                    throw e;
-                });
+              ['catch'](function(e) {
```
Comment (trentgrover-wf) [inline src/adapter.js]: this chunk still looks like 2 space indents

Comment (valotas) [inline src/adapter.js]: Let me fix that

**Comment by rmconsole2-wf** [comment]
@valotas This pull request has merge conflicts, please resolve.

**Comment by trentgrover-wf** [comment]
+1
@maxwellpeterson-wf @jayudey-wf

**Comment by rmconsole2-wf** [comment]
@valotas This pull request has merge conflicts, please resolve.

**Comment by dustinlessard-wf** [comment]
@valotas , If we intend on moving ahead with this, can you resolve the merge conflicts?

**Comment by valotas** [comment]
@dustinlessard-wf I almost forgot that.

I fixed the conflicts last time, but I don't feel like doing it every month or so :). So let me know if you feel like merging it and I'll fix them.

**Comment by evanweible-wf** [comment]
@valotas would you mind merging this PR to address the merge conflicts? https://github.com/valotas/karma-jspm/pull/1

**Comment by valotas** [comment]
@evanweible-wf I've merged your pr but there were some remaining warnings that I've fixed. It looks like this PR can be merged now. Thanks a lot for the pr anyway!

**Comment by evanweible-wf** [comment]
+1

@trentgrover-wf @maxwellpeterson-wf @dustinlessard-wf

**Comment by dustinlessard-wf** [comment]
+1

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction (derivied from conversation with Max)
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - passing CI
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into master.

**Pull request merged**